plot-api/src/commonMain/kotlin/org/jetbrains/letsPlot/frontend/NotebookFrontendContext.kt

```diff
@@ -50,6 +51,10 @@ class NotebookFrontendContext(
         return getDisplayHtml(plotBunch.toSpec())
     }
 
+    fun getHtml(figure: Figure): String {
```

Comment:
Nice to have this method in 'Figure' instead.

Reply:
But this method depends on `NotebookFrontendContext` and its behaviour is defined by `isolatedFrame`.  
May be it's better to add extensions `Figure.toHTML(): String`, `Figure.toSVG(): String`?

Reply:
Maybe we are talking about different use-cases. What is `NotebookFrontendContext.getHtml()` is intended for?

What I mean is an analog of [`to_html()`](https://lets-plot.org/python/pages/api/lets_plot.plot.core.PlotSpec.html#lets_plot.plot.core.PlotSpec.to_html) in Python where the output would likely have some external use and is configured via parameter `iframe` as needed.

Reply:
Isn't that what `ggsave()` does (almost)? It seems logical that `NotebookFrontendContext` and its methods are responsible for the internal logic associated with the notebook . And with `ggsave()` or  `toHTML()` I described user can create its own html and manipulate it somehow (including in the notebook outputs).

Reply:
But `NotebookFrontendContext` already has `getDisplayHtml()`, which has a more precise name considering its purpose. What's the need for `getHtml()`?

Reply:
`to_html()` in Python is a convenience method. Same as `ggsave()` basically but also can return html as string in case `path` is not provided.

Reply:
Wait ) Just realized there is a bunch of `getHtml()` methods in `NotebookFrontendContext` already.